pkg/storage/db.go

```diff
@@ -13,13 +13,17 @@ import (
 	"github.com/pkg/errors"
 )
 
-const tableAWSResources = "aws_resources"
-const tableAWSIPS = "aws_ips"
-const tableAWSHostnames = "aws_hostnames"
-const tableAWSEventsIPSHostnames = "aws_events_ips_hostnames"
-const createScript = "2_create.sql"
-
-const added = "ADDED" // one of the network event types we track
+const (
+	defaultPartitionInterval   = 3 // months
```

Comment:
Question: Why use an integer of months rather than a `time.Duration` value?

Reply:
there is no month duration. the golang durations go up to an hour

Reply:
Do these need to be calendar months? Is the goal to align the partitions with the calendar? For example, would it be OK to have a 90 day interval even though that won't map perfectly to calendar months?

Reply:
> Do these need to be calendar months? 

Not necessarily. There were a few things I considered during these changes:
1. the original implementation used calendar months, and this ticket was only around automating the partition creation, so I am trying to keep the scope to that
2. most reference material online uses calendar months. I am not sure of any well-documented advantages.
3. if we do need to eventually need to go to a more granular partition scheme, it is much easier to become more granular, as opposed to transitioning from every X days to every X calendar months